tools/testbuild.sh

```diff
@@ -190,7 +190,16 @@ function configure {
 function build {
   echo "  Building NuttX..."
   echo "------------------------------------------------------------------------------------"
-  makefunc ${JOPTION} ${MAKE_FLAGS} 1>/dev/null
+
+  elf=`grep CONFIG_ELF=y $nuttx/.config`
```

Comment:
I think elf line should write as elf=`grep CONFIG_ELF=y $nuttx/.config || true`  to fix  check build break as https://github.com/apache/incubator-nuttx/pull/422 blacklist does.

Reply:
@liuguo09 Thanks for the comment, I'll fix it later.